README.md

```diff
@@ -108,6 +120,10 @@ If this is set to true (default) then the migration process will transfer merge
 
 As default it is set to false. Doesn't fire the requests to github api and only does the work on the gitlab side to test for wonky cases before using up api-calls
 
+#### clearIssueAssignment
```

Comment:
I believe would be better to have two commits:
1. One commit for keepLocal+overrideURL+useSe
2. Another commit for `clearIssueAssignment`